docs/articles/actors/untyped-actor-api.md

```diff
@@ -758,6 +758,38 @@ Note that the stash is part of the ephemeral actor state, unlike the mailbox. Th
 > [!NOTE]
 > If you want to enforce that your actor can only work with an unbounded stash, then you should use the `UntypedActorWithUnboundedStash` class instead.
 
+### Bounded Stashes
```

Comment:
Copy and paste from the `ReceiveActor` page - we should probably have a separate page on behavior-switching and stashing.